maintenance/wikia/WikiFactoryVariables/migrateWikiWordmarks.php

```diff
@@ -58,23 +58,24 @@ public function execute() {
 			return false;
 		}
 
-		$this->output( "Variable: " . self::WIKI_FACTORY_VARIABLE . " (Id: $varData[cv_id])" . PHP_EOL );
-		$this->debug( "Variable data: " . json_encode( $varData ) . PHP_EOL );
+		$themeSettings = new ThemeSettings( $wgCityId );
 
-		$wg = F::app()->wg;
-		$wg->User = User::newFromName( Wikia::BOT_USER );
-		$wg->User->load();
+		if ( empty( $themeSettings ) ) {
```

Comment:
Is this check needed? The constructor for `ThemeSettings` doesn't look like it would fail, so it would always return a `ThemeSettings` object.

Reply:
Leftovers... Fix: c15a52b2e2c